src/index.html

```diff
@@ -7,12 +7,48 @@
       content="width=device-width, initial-scale=1.0"
     />
     <title>Calendar</title>
+    <link
+      rel="stylesheet"
+      href="styles/reset.css"
+    />
     <link
       rel="stylesheet"
       href="styles/index.scss"
     />
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--start-day-sun calendar--month-length-31">
```

Comment:
In this case better way use list element, because we have duplicate elements       `<div class="calendar__day"></div>`

Reply:
ok